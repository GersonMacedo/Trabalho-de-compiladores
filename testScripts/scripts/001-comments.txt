! -s LEXICAL
! Has only comments, should not trigger any lexical errors