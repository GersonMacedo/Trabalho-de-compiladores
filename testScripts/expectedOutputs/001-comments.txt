
Performing only lexical analysis at testScripts/scripts/001-comments.txt
No errors found
